### PR TITLE
Creation of Basic User Profile page

### PR DESCRIPTION
**Pull request by svaldes1**
This PR allows for the user to access basic data from their user profile. It also allows the user to update their name and email. UI and input verification will be improved with time.

### CODE REVIEW

**Review by TheAndreiUrsu — APPROVED** [review]
LGTM

**Comment by SamLonneman** [comment]
I like it :)

File: backend/api/migrations/0012_task.py [hunk 1]
```diff
@@ -33,4 +33,4 @@ class Migration(migrations.Migration):
                 ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
             ],
         ),
-    ]
+    ]
```
Comment (SamLonneman) [inline backend/api/migrations/0012_task.py:36]: Hope this works :D

**Review by SamLonneman — APPROVED** [review]
:)

**Pull request merged**